app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,272 @@
+    <string name="pref_resume_on_connect">Afspelen als headset verbonden</string>
+    <string name="pref_resume_on_connect_detail_on">Muziek zal spelen als een hoofdtelefoon verbonden wordt en Jockey aan staat.</string>
+    <string name="pref_resume_on_connect_detail_off">Muziek zal nooit automatisch spelen bij het verbinden van de hoofdtelefoon</string>
+    <string name="gesture_detail">Veeg over de albumillustratie om naar het volgende nummer te gaan, of raak aan om te pauzeren</string>
+    <string name="gesture_description">Gebaren inschakelen</string>
+    <string name="pref_sls_action_install">Installeer Simple Last.fm Scrobbler</string>
+    <string name="pref_sls_action_open">Open Simple Last.fm Scrobbler instellingen</string>
+    <string name="pref_sls_scrobble">Scrobble met Simple Last.fm Scrobbler</string>
+    <string name="pref_sls_description">Jockey kan samenwerken met Simple Last.fm Scrobbler om je afspeelhistorie online te delen. De volgende keer dat je een nummer afspeelt zal dit online worden gezet.</string>
+    <string name="pref_sls_description_not_installed">"<![CDATA[         Jockey kan je afspeelhistorie online plaatsen naar services zoals Last.fm door te integreren met         een onafhankelijke applicatie. Download Simple Last.fm Scrobbler van de         Google Play Store.         <br/><br/>         Je kan meer informatie vinden over Simple Last.fm Scrobbler vinden        <a href="https://github.com/tgwizard/sls/">op zijn website</a>.]]>     "</string>
+    <string name="pref_sls_description_enabled_but_uninstalled">"<![CDATA[        Om verder te gaan met scrobbelen naar Last.fm, Simple Last.fm Scrobbler moet geinstalleerd blijven. Je         afspeelhistorie zal dan niet worden gedeeld. Als je niet langer wil scrobbelen kan je dit uitzetten.         <br/><br/>         Jockey kan je afspeelhistorie online plaatsen naar services zoals Last.fm door te integreren met         een onafhankelijke applicatie. Download Simple Last.fm Scrobbler van de         Google Play Store.         <br/><br/>         Je kan meer informatie vinden over Simple Last.fm Scrobbler vinden        <a href="https://github.com/tgwizard/sls/">op zijn website</a>.]]>     "</string>
+    <string name="equalizer_conflict_note">Er is een andere equalizer app geinstalleerd op dit apparaat. Dit kan problemen veroorzaken met Jockey\'s equalizer. Schakel Jockey\'s equalizer uit om de systeem-wijde equalizer te gebruiken.</string>
+
+    <!-- Prefixes that are ignored when sorting titles -->
+    <string-array name="ignored_title_prefixes" tools:ignore="InconsistentArrays">
+        <item>De</item>
+        <item>Het</item>
+        <item>Een</item>
+        <item>Ten</item>
+        <item>d'</item>
+        <item>'t</item>
+        <item>'s</item>
+        <item>d' n</item>
```

Comment:
Quick question about the last four strings here. How do they appear in titles? Are they glued to other words? Or do they appear with a space afterwards?

The current logic here is to filter these out as entire words. So "The Bridge Between" gets grouped with B's, but "Theater Between" will get grouped with the T's. There's an extremely high chance that this approach breaks down in other languages and that the sorting logic needs to get updated, but that's not something we need to fix right this second in this PR.

Reply:
They all appear with a space afterwards except for `d'`.

You could say they're abbrevations, for example:
`Het Zwarte Paard` --> `'t Zwarte Paard`

and since `Het` is a prefix, we should ignore `'t` too.

Also, the circleci build has failed due to non-escpaed `'`'s, I'll repair it in the next commit.